docs/pywbemcli/cmdshelp.rst

```diff
@@ -51,6 +51,7 @@ Help text for ``pywbemcli``:
                                       --mock-server options, since each defines a
                                       WBEM server. Default: EnvVar PYWBEMCLI_NAME,
                                       or none.
+
```

Comment:
Please use Click 7.,0 in the python env where you create this file. We should not mix different things into one PR.
The transition to the changes Click 7.1 introduces (e.g. to this file) should be done in a different PR.

Reply:
The master is now usig click 7,1.1 and bypassing 7.1.

Reply:
No. Your python env is set up with Click 7.1.1. The requirements file only requires Click>=7.0, excluding 7.1, so both 7.0 and 7.1.1 are allowed. What I am suggesting is that we manually install Click 7.0 in our python envs for the time being, so that the changes driven by Click 7.1.1 are introduced separately.

Reply:
I have PR #511 under way that provides the basis to use Click 7.1.1.It is easier for this PR to wait for or rebase on that PR.

Reply:
Now I understand.